taxi/urls.py

```diff
@@ -40,9 +44,23 @@
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path(
+        "car/toggle_driver/", ToggleDriverView.as_view(), name="toggle-driver"
+    ),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("driver/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "driver/<int:pk>/update/license",
+        DriverLicenseUpdateView.as_view(),
+        name="driver-update"
```

Comment:
Consider renaming the URL pattern to 'driver-license-update' for consistency with the URL path and to clearly indicate that this is specifically for updating the driver's license.